Sources/FluentKit/Query/QueryBuilder.swift

```diff
@@ -60,6 +78,22 @@ public final class QueryBuilder<Model>
         )
     }
 
+//    @discardableResult
+//    public func join<Value>(
+//        _ field: KeyPath<Model, Parent<Value>>,
+//        alias: String? = nil
+//    ) -> Self
```

Comment:
I can't get this method to compile without a Segmentation 11 fault.

Reply:
Have you made an SR?

Reply:
I haven't yet. No time to create an MVP to produce the error yet.